Lombiq.NodeJs.Extensions/scripts/install-dev-dependencies.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs');
+const path = require('path');
+const { execSync } = require('child_process');
+
+const { handleErrorObject } = require('./handle-error');
+
+const configPath = path.resolve(__dirname, '..', 'config', 'consumer', 'package.global.json');
+const devDependencies = JSON.parse(fs.readFileSync(configPath)).devDependencies;
+const packages = Object
+    .entries(devDependencies)
+    .map(([name, version]) => `${name}@${version}`)
+    .join(' ');
+
+// Ensure each of the default development dependencies are installed. If they already are, this finishes very quickly.
+// It is necessary, because by default NPM and PNPM don't install the dependencies of local packages, see here:
+// https://docs.npmjs.com/cli/v9/configuring-npm/package-json#local-paths
+try {
+    execSync('pnpm install --save-dev ' + packages);
```

Comment:
How about we don't touch the _package.json_ file, and just install the dependencies silently, i.e. remove the `--save-dev` flag?

Reply:
If you remove the `--save-dev` flag it will still touch the package.json, just add them to the `dependencies` instead of the `devDependencies`. I think this is more appropriate.

Reply:
Concluding from [this open PNPM issue](https://github.com/pnpm/pnpm/issues/2020), there's currently no way to avoid adding installed packages to the _package.json_ and _pnpm-lock.yaml_ 😔

Reply:
I've looked into this some more, and I think we should be using `pnpm add` here instead of install. This also makes sense in the light of us calling `pnpm install` as the next step after this one. I've already successfully tried that out on my machine.

With that, I suggest to rename the script to `add-dev-dependencies.js`.

Reply:
I've already done that in the accompanying [suggestions PR](https://github.com/Lombiq/NodeJs-Extensions/pull/25/files).

Reply:
There's another nice benefit of using `pnpm add` - it uses PNPM's default version selector, which is `^` 😁 and results in having the same package `name@version` entries that we have in our NE _package.json_.